File: store/src/main/java/fooddelivery/domain/배달시작됨.java
package fooddelivery.domain;

import fooddelivery.domain.*;
import fooddelivery.infra.AbstractEvent;
import java.util.*;
import lombok.*;

@Data
@ToString
public class 배달시작됨 extends AbstractEvent {

    private Long id;
    private String 요리종류;
    private String 배달지주소;
    private String orderId;

    public 배달시작됨(주문관리 aggregate){
        super(aggregate);
    }
    public 배달시작됨(){
        super();
    }
}

File: app/배달시작됨.py

from AbstractEvent import AbstractEvent
import json

class 배달시작됨(AbstractEvent):
    id : int
    요리종류 : str
    배달지주소 : str
    orderId : str
    
    def __init__(self):
        super().__init__()
        self.id = None
        self.요리종류 = None
        self.배달지주소 = None
        self.orderId = None

